server/routes/companyRoutes.js

```diff
@@ -63,6 +63,23 @@ router.get('/profile/:companyId', (req, res) => {
     })
 })
 
+router.post('/profile/:id', (req,res) => {
+  //grabs id from the url
+  const id = req.params.id
+  const body = req.body
+  body.company_id = id
+  db.addProfile(body)
+  .then(id => {
+    //returns fully fleshed object to the display company profile page, after edit is hit
+    res.json(body)
+  })
+  .catch(err=> {
+    console.error(err)
```

Comment:
add `// eslint-disable-next-line no-console` above this line

Reply:
sorry, I don't have eslint running at the mo so I couldn't tell it was an issue